### ISSUE TITLE
Add group form labels to the Experian personal details form

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4076]
## Area or Component
Experian personal details form - sign up flow

## Quick Summary
There are group form labels coded as paragraphs ("Personal contact information" and "home address")

## Accessibility Violation
Group labels MUST be programmatically associated with the group if the individual labels for each element in the group are insufficient on their own.: https://dequeuniversity.com/class/forms/group-labels/semantic

## Action Requested
From Deque:

> Related form controls can be programmatically grouped using HTML or ARIA techniques:
> 
> The HTML solution consists of using a `<fieldset>` to semantically group the form fields and a `<legend>` to provide a semantic group label.
> The ARIA solution consists of semantically grouping fields within a group region and providing a semantic group label using the `aria-labelledby` attribute.
> 

## Screenshots (optional)
![Screen Shot 2022-07-26 at 3 20 54 PM](https://user-images.githubusercontent.com/80282552/181122263-b7003580-d681-4656-8685-0bea92020c48.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=86:87
